test_scripts/Defects/8_1/1883_2_PerformInteraction_REJECTED_for_being_deleted_InteractionChoceSet_after_resumption.lua

```diff
@@ -0,0 +1,137 @@
+-- SDL does:
+-- - transfer request to HMI
+-- 5. App sends PerformInteraction(choiceSetID_1) request
+-- SDL does:
+-- - not transfer request to HMI
+-- - responds with REJECTED to the App
+-- 6. HMI responds to DeleteInteractionChoiceSet
+-- SDL does:
+-- - responds DeleteInteractionChoiceSet(SUCCESS) to app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestVrAddCommand = {
+  cmdID = 111,
+  type = "Choice",
+  vrCommands = { "Choice111" }
+}
+
+local requestChoiceParams = {
+  interactionChoiceSetID = 1,
+  choiceSet = {
+    {
+      choiceID = 111,
+      menuName = "Choice111",
+      vrCommands = { "Choice111" }
+    }
+  }
+}
+
+local deleteChoiceSetID = {
+  interactionChoiceSetID = requestChoiceParams.interactionChoiceSetID
+}
+
+local deleteResponseVrParams = {
+  cmdID = requestChoiceParams.choiceID,
+  type = "Choice"
+}
+
+local performParams = {
+  initialText = "TextInitial",
+  interactionMode = "MANUAL_ONLY",
+  interactionChoiceSetIDList = { requestChoiceParams.interactionChoiceSetID },
+  initialPrompt = {
+    { type = "TEXT", text = "pathToFile1" }
+  }
+}
+
+local hashId = ""
+
+--[[ Local Functions ]]
+local function createInteractionChoiceSet(pChoiceParams)
+  local cid = common.mobile.getSession():SendRPC("CreateInteractionChoiceSet", pChoiceParams)
+
+  common.hmi.getConnection():ExpectRequest("VR.AddCommand", requestVrAddCommand)
+  :Do(function(_, data)
+      common.hmi.getConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+  common.mobile.getSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  common.mobile.getSession():ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      hashId = data.payload.hashID
+    end)
+end
+
+local function sendPerformInteraction_REJECTED()
+  local cid = common.mobile.getSession():SendRPC("PerformInteraction", performParams)
+  common.mobile.getSession():ExpectResponse(cid, { success = false, resultCode = "REJECTED" })
+end
+
+local function deleteInteractionChoiceSet(pDeleteChoiceParams, pDeleteVrCommandParams)
+  local cid = common.mobile.getSession():SendRPC("DeleteInteractionChoiceSet", pDeleteChoiceParams)
+  common.hmi.getConnection():ExpectRequest("VR.DeleteCommand", pDeleteVrCommandParams)
+  :Do(function(_, data)
+      sendPerformInteraction_REJECTED()
+      local function deleteCommandResp()
+        common.hmi.getConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+      end
+      common.run.runAfter(deleteCommandResp, 1000)
+    end)
+  common.mobile.getSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+local function reRegisterApp(pParams)
```

Comment:
Please remove extra variable pParams
```suggestion
local function reRegisterApp()
```

Reply:
Done